gradle.properties

```diff
@@ -23,7 +23,6 @@ googleAnalyticsId=UA-145425527-1
 
 # Gradle options
 org.gradle.jvmargs=-Xmx1280m
-org.gradle.configureondemand=true
```

Comment:
Could you explain why this has been removed?

Reply:
Oops completely forgot to explain - added to PR description